src/Modules/Content/Dnn.PersonaBar.Pages/admin/personaBar/App_LocalResources/Pages.resx

```diff
@@ -1522,4 +1516,37 @@
   <data name="ModuleCopyType.Reference" xml:space="preserve">
     <value>Reference</value>
   </data>
+  <data name="FilterByModifiedDateText.Text" xml:space="preserve">
+    <value>Filter by Modified Date Range</value>
+  </data>
+  <data name="FilterbyPageTypeText.Text" xml:space="preserve">
+    <value>Filter by Page Type</value>
+  </data>
+  <data name="FilterbyPublishStatusText.Text" xml:space="preserve">
+    <value>Filter by Publish Status</value>
+  </data>
+  <data name="lblCancel.Text" xml:space="preserve">
```

Comment:
dup key

Reply:
which one is duplicate key?

Reply:
since we get Cancel.Text already, don't need the same text again